ensembl/conf/json/parus_major_vcf.json

```diff
@@ -1,14 +1,14 @@
 {
   "collections": [
     {
-      "id": "parus_major_EVA",
-      "source_name": "EVA",
-      "description": "Variants from EVA",
+      "id": "great_tit_EVA_PRJEB24964",
+      "source_name": "PRJEB24964",
+      "description": "Variants from EVA study PRJEB24964",
       "species": "parus_major",
       "assembly": "Parus_major1.1",
       "type": "remote",
       "use_as_source" : 1,
-      "filename_template": "https://ftp.ebi.ac.uk/pub/databases/eva/rs_releases/release_3/by_species/parus_major/GCA_001522545.2/GCA_001522545.2_current_ids.vcf.gz"
+      "filename_template": "http://ftp.ebi.ac.uk/pub/databases/eva/PRJEB24964/GreatTits_PolyMonoMinorHom1.10_NoUN_NoScaffolds.vcf.gz"
```

Comment:
Hi @Mehrnaz-Charkhchi , there is no reason in particular. I am sure it would work fine even with https. We were trying to revert the code changes for this species and it seems to have http in the [previous version](https://github.com/Ensembl/public-plugins/blame/release/107/ensembl/conf/json/parus_major_vcf.json) of the code. Do you want me to switch it to https if that is something we prefer?